### ISSUE TITLE
Macos构建失败

### ISSUE THREAD

**Issue by beihaifeiwu** [alibaba/PhotonLibOS#109]
```bash
1 warning generated.
/Users/beihaifeiwu/Workspace/study/cpp/PhotonLibOS/net/http/message.cpp:244:19: error: return type of out-of-line definition of 'photon::net::http::Message::body_size' differs from that in the declaration
uint64_t Message::body_size() const {
~~~~~~~~          ^
/Users/beihaifeiwu/Workspace/study/cpp/PhotonLibOS/net/http/message.h:100:12: note: previous declaration is here
    size_t body_size() const;
    ~~~~~~ ^
1 warning generated.
/Users/beihaifeiwu/Workspace/study/cpp/PhotonLibOS/net/http/server.cpp:211:21: error: variable has incomplete type 'struct stat'
        struct stat buf;
                    ^
/Users/beihaifeiwu/Workspace/study/cpp/PhotonLibOS/include/photon/fs/filesystem.h:41:8: note: forward declaration of 'stat'
struct stat;
       ^
1 warning and 1 error generated.
make[2]: *** [CMakeFiles/photon_obj.dir/net/http/message.cpp.o] Error 1
1 warning generated.
1 warning and 2 errors generated.
make[2]: *** [CMakeFiles/photon_obj.dir/thread/thread.cpp.o] Error 1
1 warning and 1 error generated.
make[2]: *** [CMakeFiles/photon_obj.dir/net/http/server.cpp.o] Error 1
1 warning generated.
/Users/beihaifeiwu/Workspace/study/cpp/PhotonLibOS/net/http/client.cpp:31:23: warning: unused variable 'kMinimalStreamLife' [-Wunused-const-variable]
static const uint64_t kMinimalStreamLife = 300UL * 1000 * 1000;
                      ^
2 warnings generated.
2 warnings generated.
make[1]: *** [CMakeFiles/photon_obj.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by lihuiba** [comment]
Sorry for this late reply, but we were busy dealing with covid-19 and the Chinese new year. And we can work on it now.

**Comment by beef9999** [comment]
This issue has been fixed by the release of v0.5.3.

Closed.
